src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1885,15 +1885,18 @@ Expression* TranslateToFuzzReader::makeRefFuncConst(Type type) {
   }
   // We don't have a matching function, so create a null with high probability
   // if the type is nullable or otherwise create and cast a null with low
-  // probability.
-  if ((type.isNullable() && !oneIn(8)) || oneIn(8)) {
+  // probability (which would trap).
+  if ((type.isNullable() && !oneIn(16)) || oneIn(16)) {
```

Comment:
I would keep the probability of using a `ref.null` for a nullable reference low, too. It's more interesting to have a non-null value for a nullable reference than a null value for a non-null reference.

Reply:
Ok, I adjusted that probability now.

I also made us not trap if no value is returned - we can just have a nop there instead of an unreachable.